src/Analyzers/Core/Analyzers/Helpers/DiagnosticHelper.cs

```diff
@@ -265,6 +265,12 @@ static string EncodeIndices(IEnumerable<int> indices, int additionalLocationsLen
             if (id.StartsWith("JSON", StringComparison.Ordinal))
                 return null;
 
+            if (id == RemoveUnnecessaryImports.AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer.DiagnosticFixableId)
+                return null;
+
+            if (id == "IDE0005_gen")
```

Comment:
Add a comment for both these cases?

Reply:
I was just preserving the original behavior, and have no idea why/if this behavior is correct.

Can help link for `IDE0005_gen` be visible? The diagnostic is hidden and not configurable, so help link can never be shown?

Same applies to `AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer.DiagnosticFixableId`.

Reply:
In that case, can we return null for both cases?

Reply:
Probably add a comment on the same lines as your previous comment: `This diagnostic is hidden and not configurable, so help link can never be shown and is not applicable`